sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -62,12 +63,14 @@ def __init__(self, endpoint, credential, **kwargs):
         authentication_policy = AzureKeyCredentialPolicy(
             name=COGNITIVE_KEY_HEADER, credential=credential
         )
+
         self._client = _BatchDocumentTranslationClient(
             endpoint=endpoint,
             credential=credential,  # type: ignore
             api_version=self._api_version,
             sdk_moniker=USER_AGENT,
             authentication_policy=authentication_policy,
+            http_logging_policy=get_http_logging_policy(),
```

Comment:
asking as I don't know how this works on Python... if the service introduces new headers/query parameters, this needs to be updated? or will the user be able to add more logging policies if they desire? -> it might not be needed if the list is always updated though

Reply:
There is a list of allowed headers in the policy, but it's general and doesn't include service-specific headers or query params. It can always be updated if the service starts sending anything new.